src/Bicep.Cli.IntegrationTests/TestFrameworkCommandTests.cs

```diff
@@ -64,6 +64,114 @@ public async Task Test_NonBicepFiles_ShouldFail_WithExpectedErrorMessage()
             }
         }
 
+        [TestMethod]
+        public async Task Test_command_ShouldSucceed()
```

Comment:
Would you mind breaking the individual test cases into separate test methods? E.g.:

`public async Task Test_command_should_succeed_basic()`
`public async Task Test_command_should_succeed_when_passing_required_parameter()`
`public async Task Test_command_should_succeed_when_all_assertions_pass()`